Tests/CommandsCoreTests/TestHelper.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  TestHelper.swift
+//  CommandsCoreTests
+//
+//  Created by Martin Kim Dung-Pham on 17.10.17.
+//
+
+import Foundation
+
+extension String {
+    func makeScript(for type: AnyObject.Type) -> String? {
+        let bundle = Bundle(for: type)
+        if let url = URL(string: bundle.resourcePath! + "/script.sh") {
```

Comment:
What about an early escape with a `guard`? And why would you force unwrap if you're `if let`ting anyway?

What about:

```swift
guard let resourcePath = bundle.resourcePath, let url = URL(string: "\(resourcePath)/script.sh") else { return nil }
```

The interpolation (instead of concatenation) of the strings will make a tiny bit more performant as well.